docs/readthedocs/usersguide.md

```diff
@@ -1,224 +1,213 @@
+> **_NOTE:_**
+The external simulation is loaded via class loader. Thus, SIMONA and the external simulation should be built using the same JDK.
+
+### Requirements
 
-Requirements
-------------
 
 **External simulation**
 
 - The external simulation should be implemented in its own project (repository).
 - The project should include the *shadowJar* gradle plugin (``id "com.github.johnrengelman.shadow" version "x.y.z"``).
-- A class (called *main class* here) needs to extend ``edu.ie3.simona.api.schedule.ExtSimulation`` and thus implement the two methods ``List<Long> initialize()´´ and ``List<Long> doActivity(long tick)``. The method ``initialize´´ is called when the external simulation needs to be initialized whereas the method ``doActivity´´ is called when time step ``tick`` is triggered. ``initialize´´ and ``doActivity`` must return a list of subsequent new ticks that the sub simulation should be scheduled at.
+- A class (called *main class* here) needs to extend ``edu.ie3.simona.api.schedule.ExtSimulation`` and thus implement the two methods ``List<Long> initialize()`` and ``List<Long> doActivity(long tick)``. The method ``initialize`` is called when the external simulation needs to be initialized whereas the method ``doActivity`` is called when time step ``tick`` is triggered. ``initialize`` and ``doActivity`` must return a list of subsequent new ticks that the sub simulation should be scheduled at.
 - For each data stream, a sub-interface of ``edu.ie3.simona.api.data.ExtDataSimulation`` needs to be implemented, such as ``edu.ie3.simona.api.data.ev.ExtEvSimulation``, and all methods of the interface have to be implemented. The *main class* could be the implementing class here.
-- In order for SIMONA to recognize the external simulation, a class ``edu.ie3.simona.api.ExtLink`` has to reside inside the project. This class has to extend ``edu.ie3.simona.api.ExtLinkInterface`` and implement the corresponding methods by returning the control stream and data stream implementations (could all be the same *main class*).
+- In order for SIMONA to use the external simulation, a class that extends ``edu.ie3.simona.api.ExtLinkInterface`` has to reside inside the project. The class has to implement the corresponding methods by returning the control stream and data stream implementations (could all be the same *main class*).
+- When loading the external simulations, SIMONA is looking for the corresponding service files of the class ``edu.ie3.simona.api.ExtLinkInterface``. Therefor every external simulation needs the following service file: ``src/main/resources/META-INF/services/edu.ie3.simona.api.ExtLinkInterface``. The service file needs to contain the relative path to the class that extends ``edu.ie3.simona.api.ExtLinkInterface``.
+- An example for an external simulation can be found [here](https://github.com/ie3-institute/ExtSimSample)
```

Comment:
Would you mind adding "A *very simple* example", since this example project is only that? :)